AsyncDisplayKit/Private/_ASHierarchyChangeSet.h

```diff
@@ -128,6 +127,9 @@ NSString *NSStringFromASHierarchyChangeType(_ASHierarchyChangeType changeType);
 - (NSUInteger)newSectionForOldSection:(NSUInteger)oldSection;
 
 @property (nonatomic, readonly) BOOL completed;
+/// Whether or not changes should be animated.
+@property (nonatomic, readwrite) BOOL animated;
```

Comment:
Good call, putting this flag here. My reckoning here is, if any update in the stack is non-animated, the whole update should be non-animated. Would be fine as a follow-up task or discussion, or implement it here.